src/js/views/patients/widgets/README.md

```diff
@@ -47,11 +47,11 @@ Deprecated:
 
 Widget type `groups` has been deprecated for `workspaces`.
 
-The following widget with a `widget_type: "workspaces"`:
+The following widget with a `category: "workspaces"`:
 
 ```json
   {
-    "widget_type": "workspaces",
+    "category": "workspaces",
     "id": "workspaces",
     "definition": {
       "display_name": "Display Name"
```

Comment:
FWIW, widgets do have a `name` that is independent of `slug` now, so we probably don't need to have display name any more. That being said, the default `name` is the value of `slug`, so there would need to be coordinated clean up and/or migration.

Reply:
I thought about that, but display name is an optional label that gets added in the sidebar.. And really you could utilize the same widget differently with different display names